Lets Do This/Controllers/Base/LDTWebViewController.m

```diff
@@ -0,0 +1,49 @@
+//
+//  LDTWebViewController.m
+//  Lets Do This
+//
+//  Created by Aaron Schachter on 5/16/16.
+//  Copyright © 2016 Do Something. All rights reserved.
+//
+
+#import "LDTWebViewController.h"
+#import "GAI+LDT.h"
+
+@interface LDTWebViewController () <UIWebViewDelegate>
+
+@property (strong, nonatomic) NSString *navigationTitle;
+@property (strong, nonatomic) NSURL *webViewURL;
+
+@end
+
+@implementation LDTWebViewController
+
+
+#pragma mark - NSObject
+
+- (instancetype)initWithWebViewURL:(NSURL *)webViewURL title:(NSString *)navigationTitle{
+    self = [super init];
+    
+    if (self) {
+        _navigationTitle = navigationTitle.uppercaseString;
+        _webViewURL = webViewURL;
+    }
+    return self;
+}
+
+#pragma mark - UIViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.title = self.navigationTitle;
+    UIWebView *webView = [[UIWebView alloc] initWithFrame:self.view.frame];
+    webView.delegate = self;
+    NSURLRequest *urlRequest = [NSURLRequest requestWithURL:self.webViewURL];
+    [webView loadRequest:urlRequest];
```

Comment:
@lkpttn To answer your [question about how the UIWebView is loading content](https://github.com/DoSomething/LetsDoThis-iOS/pull/1015#issuecomment-219717936) -- this `loadRequest` method is loading the attachment's URL.

What's nice about this approach is we don't have to leave the app... but what feels a little lame in this specific example is not being able to download PDF to device for printing the template.  I've tested on device -- you are able to zoom in (now that #1016 is merged) 

![pdf](https://cloud.githubusercontent.com/assets/1236811/15328582/415a8b0c-1c0a-11e6-9484-b6a2658f0f6f.gif)